Cilsil/Log.cs

```diff
@@ -47,6 +53,29 @@ public static void RecordUnknownInstruction(string instruction)
             }
         }
 
+        /// <summary>
+        /// TODO: use https://nlog-project.org or log4net instead of this class.
+        /// </summary>
+        public static void RecordMethodElapseTime(MethodDefinition method, long elapseTime)
```

Comment:
Where does the measured time show up? In logs or printed on the output screen?

Reply:
In logs with `--outelapsetime` flag to specify path